src/Filament/Resources/PageResource.php

```diff
@@ -216,6 +216,11 @@ public static function getNavigationLabel(): string
         return __('Pages');
     }
 
+    public static function getNavigationBadge(): ?string
+    {
+        return (string) static::getEloquentQuery()->page()->count();
```

Comment:
One nit-pick - the current `page` and `posts` scopes will also scope based on `published_at` `<=` now. Which is probably a great default for 80% of times you want to scope these.

However, in the backend of the system you may not want that applied as that will exclude items to be posted in the future. And in the context of the filament panel badge number it may be best to have the "real" total of posts/pages, even ones to be published in the future included in that total.